src/FffGcodeWriter.cpp

```diff
@@ -817,6 +817,17 @@ void FffGcodeWriter::addMeshLayerToGCode(SliceDataStorage& storage, SliceMeshSto
     }
     part_order_optimizer.optimize();
 
+    EFillMethod infill_pattern = mesh->getSettingAsFillMethod("infill_pattern");
+    if ((infill_pattern == EFillMethod::LINES || infill_pattern == EFillMethod::ZIG_ZAG) && mesh->infill_angles.size() == 0)
```

Comment:
The infill angles can be used for any linear infill type: lines, grid, cubic, cubicsubdiv, tetrahedral, triangles and zig_zag